src/Extractor/HttpClient.php

```diff
@@ -222,8 +222,8 @@ public function fetch($url, $skipTypeVerification = false, $httpHeader = [])
         // remove utm parameters & fragment
         $uri = new Uri(str_replace('&amp;', '&', $effectiveUrl));
         parse_str($uri->getQuery(), $query);
-        $queryParameters = array_filter($query, fn ($k) => !(0 === stripos($k, 'utm_')), \ARRAY_FILTER_USE_KEY);
-        $effectiveUrl = (string) Uri::withQueryValues(new Uri($uri->withFragment('')->withQuery('')), $queryParameters);
+        $queryParameters = array_filter($query, fn ($k) => !(0 === stripos((string) $k, 'utm_')), \ARRAY_FILTER_USE_KEY);
+        $effectiveUrl = (string) new Uri($uri->withFragment('')->withQuery(http_build_query($queryParameters)));
```

Comment:
Note that `http_build_query` is not an inverse of `parse_str`:

```php
php > parse_str('foo[]=x', $query); var_dump(http_build_query($query));
string(12) "foo%5B0%5D=x"
```

Reply:
In selfoss, I have decided to just use `explode` for this purpose:

https://github.com/fossar/selfoss/blob/0d7bde56e502f7d79bfb38dcdd657c7da89cf1f1/src/spouts/rss/fulltextrss.php#L120